src/uwtools/tests/drivers/test_driver.py

```diff
@@ -423,6 +442,45 @@ def test_Driver_runscript(arg, driverobj, type_):
         assert isinstance(runscript.call_args.kwargs[arg], type_)
 
 
+def test_driver_show_output(capsys, config):
+    ConcreteDriverTimeInvariant(config).show_output()
+    expected = """
+    {
+      "bar": [
+        "/path/to/bar1",
+        "/path/to/bar2"
+      ],
+      "foo": "/path/to/foo"
+    }
+    """
+    assert capsys.readouterr().out.strip() == dedent(expected).strip()
```

Comment:
A test of the kind of output we'd expect to see on `stdout` if a CLI command like `uw <driver> show_output ...` were issued.